.github/workflows/lint-code.yml

```diff
@@ -7,18 +7,19 @@ on:
       - reopened
       - synchronize
   push:
-    branches:
-      - master
-      - "v[0-9]+.[0-9]+"
     tags:
-      - "(dev-)?v[0-9]+.[0-9]+.[0-9]+"
+      - dev-v[0-9]+.[0-9]+.[0-9]+
 
 jobs:
   define-versions:
     runs-on: ubuntu-latest
     outputs:
       pyVersions: '["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]'
 
+    steps:
+      - name: Placeholder
+        run: echo "Placeholder"
```

Comment:
do we need this step?

Reply:
Removed this. It was an artifact from setting up a large pipeline with `workflow_call` triggers

Reply:
Added this back. Workflows need atleast a single step to run.
They error out silently otherwise, and was missed.